### ISSUE TITLE
UI can load with a progenitor

### ISSUE THREAD

**Issue by nick-stebbings** [joshuavial/bridge.hc#8]
I have scaffolded basic ui and am now realising the chicken-egg situation with initialising the bridge zome.

The only way to do this with a progenitor in the properties is to use the `hc cli`. I posted onto discord dev.hc and I believe Guillem has made a feature request so that this is customisable at launch within the launcher. 

I may as well issue a pull request for the ui and work on the transactions log. You can run it and see the error that comes up with regards to no AuthorityList having been instantiated.

**Comment by joshuavial** [comment]
yeah, I see the chicken and egg - I reckon we just keep building and see if we get a steer from the devs about the best way to instantiate the local app
